_i18n/en.yml

```diff
@@ -249,6 +249,10 @@ downloads:
   desktop: Desktop
   antivirus: Antivirus
   sourcearchive: Source Code (archive)
+  localsync: Local synchronization
+  localsyncinfo: These wallets scan the blockchain on your device and don't share any key with remote servers. They provide higher privacy, but require more time synchronizing.
+  remotesync: Remote synchronization (lightweight)
+  remotesyncinfo: These wallets share your private view key with a remote server, which scans the blockchain for you. They are faster to use, but your privacy can potentially be compromised if you use a third-party server.
```

Comment:
```suggestion
  remotesyncinfo: These wallets share your private @view-key (needed to see incoming transactions) with a remote server, which scans @blocks looking for your transactions. They are faster to use because they continuously scan the blockchain looking for transactions, but your privacy can be lessened if you don't control the remote server.
```

Reply:
I added most of your suggestion, but made a shorter version.